From cfa695f044e7676fd86ed2940b4877cf43d87814 Mon Sep 17 00:00:00 2001
From: Fedora dracut team <dracut-maint@redhat.com>
Date: Mon, 13 Jan 2014 11:26:04 -0500
Subject: [PATCH 67/67] udev: install seat rules

plymouth needs these rules to know when the graphics hardware is
ready for scan out.
---
 modules.d/95udev-rules/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/95udev-rules/module-setup.sh b/modules.d/95udev-rules/module-setup.sh
index 2cdcc1b..094479a 100755
--- a/modules.d/95udev-rules/module-setup.sh
+++ b/modules.d/95udev-rules/module-setup.sh
@@ -4,60 +4,61 @@
 
 install() {
     local _i
 
     # Fixme: would be nice if we didn't have to know which rules to grab....
     # ultimately, /lib/initramfs/rules.d or somesuch which includes links/copies
     # of the rules we want so that we just copy those in would be best
     inst_multiple udevadm cat uname blkid \
         /etc/udev/udev.conf
 
     [ -d ${initdir}/$systemdutildir ] || mkdir -p ${initdir}/$systemdutildir
     for _i in ${systemdutildir}/systemd-udevd ${udevdir}/udevd /sbin/udevd; do
         [ -x "$_i" ] || continue
         inst "$_i"
 
         if ! [[ -f  ${initdir}${systemdutildir}/systemd-udevd ]]; then
             ln -fs "$_i" ${initdir}${systemdutildir}/systemd-udevd
         fi
         break
     done
     if ! [[ -e ${initdir}${systemdutildir}/systemd-udevd ]]; then
         derror "Cannot find [systemd-]udevd binary!"
         exit 1
     fi
 
     inst_rules 50-udev-default.rules 60-persistent-storage.rules \
         61-persistent-storage-edd.rules 80-drivers.rules 95-udev-late.rules \
         60-pcmcia.rules \
         50-udev.rules 95-late.rules \
         50-firmware.rules \
+        70-uaccess.rules 71-seat.rules 73-seat-late.rules \
         75-net-description.rules 80-net-name-slot.rules \
         "$moddir/59-persistent-storage.rules" \
         "$moddir/61-persistent-storage.rules"
 
     prepare_udev_rules 59-persistent-storage.rules 61-persistent-storage.rules
     # debian udev rules
     inst_rules 91-permissions.rules
 
     {
         for i in cdrom tape dialout floppy; do
             if ! egrep -q "^$i:" "$initdir/etc/group" 2>/dev/null; then
                 if ! egrep "^$i:" /etc/group 2>/dev/null; then
                         case $i in
                             cdrom)   echo "$i:x:11:";;
                             dialout) echo "$i:x:18:";;
                             floppy)  echo "$i:x:19:";;
                             tape)    echo "$i:x:33:";;
                         esac
                 fi
             fi
         done
     } >> "$initdir/etc/group"
 
     inst_multiple -o \
         ${udevdir}/ata_id \
         ${udevdir}/cdrom_id \
         ${udevdir}/create_floppy_devices \
         ${udevdir}/edd_id \
         ${udevdir}/firmware.sh \
         ${udevdir}/firmware \
-- 
1.8.3.1

